Line_0 $$ Entry-->Line_1 $$ public class func [ CD ]
Line_1 $$ public class func -->Line_2 $$ public void _failOver(String serverHost, int serverPort, String databasePath, String dbSubPath, String replicatedDb) [ CD ]
Line_2 $$ public void _failOver(String serverHost, int serverPort, String databasePath, String dbSubPath, String replicatedDb) -->Line_3 $$ util.DEBUG(connectionURL)[ CD ]
Line_2 $$ public void _failOver(String serverHost, int serverPort, String databasePath, String dbSubPath, String replicatedDb) -->Line_4 $$ Connection conn = DriverManager.getConnection(connectionURL)[ CD ]
Line_4 $$ Connection conn = DriverManager.getConnection(connectionURL)-->Line_4 $$ Connection conn = DriverManager.getConnection(connectionURL)[ CD ]
Line_2 $$ public void _failOver(String serverHost, int serverPort, String databasePath, String dbSubPath, String replicatedDb) -->Line_5 $$ util.DEBUG("failOver Unexpectedly connected as: " + connectionURL)[ CD ]
